homeassistant/components/bme280/__init__.py

```diff
@@ -89,7 +91,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-async def async_setup(hass, config):
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
```

Comment:
This falls under [ADR19](https://github.com/home-assistant/architecture/blob/master/adr/0019-GPIO.md), should I remove this change, or is it OK to include it in "bulk cleanup"?

Reply:
Nah its ok for now.